FreeRTOS/FreeRTOS
# Add RISC-V demo for the spike simulator.

Add RISC-V demo for the spike simulator.
-----------

Started with RISC-V-Qemu-virt_GCC and changed it until it worked. It currently relies on an implementation detail in spike for I/O, but I'm looking into making that official.

Test Steps
-----------
Manually ran the binary using the spike simulator, and did some minimal debugging with gdb/OpenOCD.

Related Issue
-----------
None.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
